tests/console/system_state.pm

```diff
@@ -24,6 +33,8 @@ sub run {
     check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
     script_run "ps axf > /tmp/psaxf.log";
     script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
+    assert_script_run "tar cvf /tmp/info_during_installation.tar /tmp/psaxf.log /tmp/loadavg_consoletest_setup.txt";
```

Comment:
We could reuse `tar_and_upload_log` here from `lib\opensusebasetest`.

Reply:
sure why not. let me test it.

Reply:
Due to network issues i havent managed to test it. although it will require some extra changes. if you think that it is important to use the `tar_and_upload_log` we have to wait.

Reply:
OK VR using `tar_and_upload_log` http://aquarius.suse.cz/tests/3435#downloads